charts/metrics-agent/values.yaml

```diff
@@ -90,5 +90,11 @@ podLabels: {}
 # Extra labels to add to all resources, including pods.
 additionalLabels: {}
 
+# Extra annotations to add to the pod only.
+podAnnotations: {}
+
+# when toggling to true, the proper volumeMount must be configured correctly otherwise the agent will crash
```

Comment:
Any docs we need to provide around this regarding where to add the mount?

Reply:
added more context to values.yaml